NSS-Day-Cohort-42/rare-media-moguls
# Ap styling

#### Changes Made
1. Modified styling in for login page, posts list (individual posts in post list still need work), tag manager, and category manager
2. Changed headings and arrangement of elements on above components to more closely match wireframe

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout ap-styling
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> Log out, view the login page, and then log back in
> Check styling on the post list page, category & tag managers
5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
